src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -44,22 +52,36 @@
 import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.EVIDENCE_HANDLED;
 import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.SCANNED_DOCUMENTS;
 import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.SEARCH_CASE_REFERENCE_TYPE;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.domains.envelopes.model.Classification.SUPPLEMENTARY_EVIDENCE_WITH_OCR;
 
 @Service
 public class AttachCaseCallbackService {
 
-    public static final String INTERNAL_ERROR_MSG = "An error occurred";
     public static final String PAYMENT_ERROR_MSG = "Payment references cannot be processed. Please try again later";
 
     private static final Logger log = LoggerFactory.getLogger(AttachCaseCallbackService.class);
 
+    private final ServiceConfigProvider serviceConfigProvider;
+
     private final CcdApi ccdApi;
 
+    private final ExceptionRecordValidator exceptionRecordValidator;
+
+    private final CcdCaseUpdater ccdCaseUpdater;
+
     private final PaymentsProcessor paymentsProcessor;
 
-    public AttachCaseCallbackService(CcdApi ccdApi, PaymentsProcessor paymentsProcessor) {
+    public AttachCaseCallbackService(ServiceConfigProvider serviceConfigProvider,
+                                     CcdApi ccdApi,
+                                     ExceptionRecordValidator exceptionRecordValidator,
+                                     CcdCaseUpdater ccdCaseUpdater,
+                                     PaymentsProcessor paymentsProcessor
+    ) {
```

Comment:
We should, at some point (not this PR), think about how to reduce the number of dependencies in this class.

We have two high-level clients of CCD - `CcdApi` and `CcdCaseUpdater`. This seems unnecessary and the distinction between them is unclear, as both can update a CCD case.